content/en/docs/Monitoring/opentelemetry-integration.md

```diff
@@ -0,0 +1,282 @@
+---
+title: Opentelemetry Integration
+description: Kyverno is instrumented with the Opentelemetry SDK. This helps us currently in exposing metrics and traces to an Opentelemetry Collector. This guide will show you how to set up a simple Opentelemetry Collector as a deployment and export Kyverno Metrics and Traces to it. This is just an example and must not be used in production.
+weight: 65
+---
+
+## Opentelemetry Setup
+
+We will work with a few yaml files are configurations to setup our collector in the kyverno namespace. The required configurations are listed below.
+
+### Install Cert-Manager
+
+```
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.9.1/cert-manager.yaml
+```
+
+### Config file for Opentelemetry Collector
+
+Create a ```configmap.yaml``` with the following content:
```

Comment:
In-line code formatting such as a file name should only be wrapped in single back ticks, not three. Use of three is for a code block.

Reply:
New lines should separate all code blocks.